ci-operator/step-registry/ipi/gcp/private/ipi-gcp-private-workflow.yaml

```diff
@@ -0,0 +1,15 @@
+workflow:
```

Comment:
If we already have ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/private/cucushift-installer-rehearse-gcp-ipi-private-workflow.yaml in this PR, we do not need this file here, right?

Reply:
I think rehearse is just temporary workflow, and the in-place workflows are there for onward references. Right?

Reply:
Productivity team will integrate the whole testing including pre, e2e, and post. We have an agreement that installer team will put all the pre/post workflow in rehearse folder.